Source/PartnerCenter.CustomerPortal/BusinessLogic/Commerce/PaymentGateways/PayU/PayUTxnStatusResponse.cs

```diff
@@ -0,0 +1,41 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="PayUTxnStatusResponse.cs" company="Microsoft">
+//      Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.Store.PartnerCenter.CustomerPortal.BusinessLogic.Commerce.PaymentGateways
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Transaction status response class
+    /// </summary>
+    public class PayUTxnStatusResponse
```

Comment:
Would it be possible to complete the XML comments? It would be helpful to see something like Gets or sets the status code instead of just Gets or sets

Reply:
Still active. 

---
In reply to: [178308676](https://github.com/PartnerCenterSamples/Reseller-Web-Application/pull/61#discussion_r178308676) [](ancestors = 178308676)